src/main.c

```diff
@@ -217,6 +242,30 @@ int exec_cmdline_arg(int argcin, char *argvin[])
 			case 'x':
 				action = ACTION_UNEXPORT_NVMET;
 				break;
+			case 'y':
```

Comment:
Same here. It is much easier to follow if in alphabetical order (capital first, then lower case).